src/sys/socket/consts.rs

```diff
@@ -178,7 +178,10 @@ mod os {
     pub const SO_REUSEPORT: c_int           = libc::SO_REUSEPORT;
     pub const SO_REUSESHAREUID: c_int       = 0x1025;
     pub const SO_SNDBUF: c_int              = libc::SO_SNDBUF;
+    #[cfg(not(target_os = "openbsd"))]
     pub const SO_TIMESTAMP: c_int           = libc::SO_TIMESTAMP;
+    #[cfg(target_os = "openbsd")]
+    pub const SO_TIMESTAMP: c_int           = 0x0800;
```

Comment:
We don't hardcode values in `nix` anymore, this needs to be upstreamed into [libc](https://github.com/rust-lang/libc) before it can be included here.